docs/getting_started/installation.md

```diff
@@ -15,7 +15,7 @@ In order to run shift inside Maya, you need to run the following inside the Maya
 ```python
 import sys
 sys.path.append("<path_to_your_shift_folder>")
-sys.path.append("<path_to_your_shift_folder>/thirdparty/python/Lib/site-packages")
+sys.path.append("<path_to_your_shift_folder>/shift/thirdparty/python/Lib/site-packages")
```

Comment:
I just realized we are using `you`!
Should we change it to `path_to_the_shift_installation_folder`?

```suggestion
sys.path.append("<path_to_your_shift_folder>/shift/thirdparty/python/Lib/site-packages")
```

Reply:
it's ok, but we should check the rest of other PRs, I can replace this but maybe I will require to revie the PR again. this will change more lines that the PR is currently changing

Reply:
it's done